deployments/container/Dockerfile

```diff
@@ -34,14 +34,35 @@ ARG GOLANG_VERSION=x.x.x
 ARG BUILDARCH
 ARG TARGETARCH
 
+# Install dependencies for Go build. Do not clear apt cache (does not
+# leak into prod stage).
 RUN apt-get update && \
     apt-get install -y \
         wget \
         make \
         git \
         gcc-aarch64-linux-gnu \
-        gcc && \
-    rm -rf /var/lib/apt/lists/*
+        gcc
+
+# Install dependencies for `bash-static` build.
+RUN apt-get install -y gpg curl autoconf
+
+# Build static bash binary (against musl).
+WORKDIR /bashbuild
+RUN git clone https://github.com/robxu9/bash-static/
+RUN ARCH="$TARGETARCH" && \
+    [ "$ARCH" = "arm64" ] && ARCH="aarch64" || true && \
+    [ "$ARCH" = "amd64" ] && ARCH="x86_64" || true && \
+    echo "detected arch: $ARCH" && \
+    cd bash-static && git checkout 021f5f29f665c92ca16a369d9f27e288c3aed0c6 && \
+    sed -i 's|https://ftp\.gnu\.org/gnu|https://ftpmirror.gnu.org/|g' ./build.sh && \
+    sed -i 's/-sLO/-sSfLO --retry 300 --connect-timeout 20 --retry-delay 2/g' ./build.sh && \
```

Comment:
The `-S` curl option (--show-errors) is critical to not waste people's time :).

Also opened https://github.com/robxu9/bash-static/pull/56.

`ftp.gnu.org` has issues right now (and probably frequently has). I looked `https://ftpmirror.gnu.org` up as a more robust alternative (CDN-like properties).

Added the explicit connect timeout and some retries as general robustness strategy (should always do that in CI).